django/contrib/admin/sites.py

```diff
@@ -106,7 +107,16 @@ def register(self, model_or_iterable, admin_class=None, **options):
                 )
 
             if model in self._registry:
-                raise AlreadyRegistered('The model %s is already registered' % model.__name__)
+                registered_admin = str(self._registry[model])
+                # Remove `.ModelAdmin` from the end for the models which is already
+                # registered without a `ModelAdmin` subclass
+                registered_admin = re.sub('.ModelAdmin$', '', registered_admin)
```

Comment:
I'd use `rstrip('.ModelAdmin')` rather than a regex.

Reply:
I think we can use `rstip()` because based on the [rstrip doc](https://docs.python.org/3.5/library/stdtypes.html#str.rstrip) 

> The chars argument is a string specifying the set of characters to be removed

```
>>> 'admin_registratiom.ModelAdmin'.rstrip('.ModelAdmin')
'admin_registrat'
```

I keep the `re.sub()`, if you prefer to don't use it we can replace it with:
```
>>> 'admin_registratiom.ModelAdmin'.split('.ModelAdmin')[0]
'admin_registratiom'
```